swift/ExampleLibrary/Sources/ExampleLibraryObjC/SomeApiObjC.mm

```diff
@@ -0,0 +1,58 @@
+#include <example_library/some_api.h>
+
+#import "SomeApiObjC.h"
+
+@implementation SomeApiObjC {
+    SomeApi someApi;
+}
+
+- (NSString *) str_cpp_to_objc:(std::string&)cpp_string {
```

Comment:
It is more common to use camel case for obj-c function, classes, properties names.